app/views/foreman/job_templates/install_errata_by_search_query_-_katello_ansible_default.erb

```diff
@@ -21,6 +21,34 @@ template_inputs:
 <%= render_template('Run Command - Ansible Default', :command => "zypper -n install -t patch #{advisory_ids.join(' ')}") %>
```

Comment:
The zypper problem already exists if you try to update suse and RHEL machines at the same time, so I'm not fixing it here.